docs-kits/kits/ESS-Kit/page_adoption-view.md

```diff
@@ -163,13 +171,179 @@ The data model follows the Catena-X Standard CX-0113 (<https://catena-x.net/de/s
 
 ![ESS_KitBusinessArchitecture](@site/static/kits/ESS-Kit/CatenaX_ESS_KitBusinessArchitecture.png)
 
-### Access & Usage Policies / Code of conduct
+### Access & Usage Policies / Code of Conduct
+
+#### Preamble
+
+**Code of Conduct of the Catena-X Use Case Sustainability / Environmental and Social standards:**
+
+In the Catena-X use case sustainability / environmental and social standards, principles are established with the aim of establishing transparent and trustworthy practices within the Catena-X network.
+These principles are based on national and international legal regulations and are aligned with internationally recognised standards such as the International Bill of Human Rights, the UN Guiding Principles on Business and Human Rights, the OECD Guidelines for Multinational Enterprises, as well as the ILO core labour standards and the principles of the UN Global Compact.
+Accordingly, this document defines the minimum requirements and clear expectations for the corporate due diligence obligations of the members. We expect you, as our member, to ensure that these requirements and expectations are also passed on appropriately to your downstream business partners. Acting responsibly and lawfully is therefore an integral part of participating in the Catena-X network.
+The Catena-X network includes companies and organizations that design, manufacture, market or provide goods and services for the automotive industry. The Code of Conduct must be adopted by every company participating in the Catena-X Use Case Sustainability / Environmental and Social Standards in a binding manner or guaranteed by an appropriate standard of its own. The recognition of the Catena-X Code of Conduct is the basis of a cooperative business relationship with our members.  Given the complexity and dynamics of the n-tier network, we rely on joint activities with all our members to identify risks to the environment and society, create more transparency and achieve more effectiveness.
```

Comment:
This is a normative statement in Tractus-X - I personally would not put it here, as governance for the network is in the hands of the Catena-X Association - as quick fix, a contextual statement would make sense (code of conduct currently in the works in the CX Association, here is the planned content)

Reply:
my last info was that we do not have another place where we can put the Code of Conduct, so we wanted to place it here and mention it in section Access & Usage Policies